test/conformance/gateway_conformance_test.go

```diff
@@ -90,9 +91,23 @@ func TestGatewayConformance(t *testing.T) {
 		}
 	}
 
+	// these tests are temporarily disabled to be able to bump the Gateway API to 0.6
+	// https://github.com/Kong/kubernetes-ingress-controller/issues/3305
+	skipTests := map[string]struct{}{
+		tests.GatewayInvalidRouteKind.ShortName:                   {},
+		tests.GatewayInvalidTLSConfiguration.ShortName:            {},
+		tests.GatewaySecretReferenceGrantAllInNamespace.ShortName: {},
+		tests.GatewaySecretReferenceGrantSpecific.ShortName:       {},
+		tests.HTTPRouteHeaderMatching.ShortName:                   {},
+		tests.HTTPRouteObservedGenerationBump.ShortName:           {},
+		tests.HTTPRouteRequestRedirect.ShortName:                  {},
```

Comment:
Actually, now that I think about it: can we explicitly mark those as skipped so that we get those logged in the output? I believe this will be better for visibility. I'm not saying that we _will_ forget about them but it will be there to annoy us until we fix it. WDYT?

Reply:
I'm gonna work on them once this PR gets merged. The idea is that they will be re-enabled very soon. Anyway, why not, I'm gonna add a log line

Reply:
If you meant marking them as skipped in the conformance test suite, such functionality has been recently added upstream, `v0.6.0` does not expose a standard way to opt out of test cases.

Reply:
I meant that we could add a line inside the subtest

```
t.Run(tt.Description, func(t *testing.T) { tt.Run(t, cSuite) })
```

which would check `if _, ok := skipTests[tt.ShortName]; ok {` and if that's true then `t.Skip()` the subtest that matched the skipped tests map that you introduced. That makes sense?

Reply:
sure, I accepted your suggestion